server/sonar-server/src/main/java/org/sonar/server/issue/IssueUpdater.java

```diff
@@ -269,14 +273,14 @@ public boolean setEffortToFix(DefaultIssue issue, @Nullable Double d, IssueChang
 
   public boolean setPastEffortToFix(DefaultIssue issue, @Nullable Double previousEffort, IssueChangeContext context) {
     Double currentEffort = issue.effortToFix();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "effortToFix"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)